omr.rc.in

```diff
@@ -0,0 +1,29 @@
+
+#include <windows.h>
+#include <winver.h>
+
+VS_VERSION_INFO VERSIONINFO
+ FILEVERSION 2,9,5,19691
+ PRODUCTVERSION 2,9,5,19691
+ FILEFLAGSMASK 0x3fL
+ FILEFLAGS 0x0L
+ FILEOS VOS_NT_WINDOWS32
+ FILETYPE VFT_DLL
+ FILESUBTYPE 0x0L
+BEGIN
+	BLOCK "StringFileInfo"
+	BEGIN
+		BLOCK "040904b0"
+		BEGIN
+			VALUE "CompanyName", "International Business Machines Corporation\0"
```

Comment:
@mstoodle ok that was my thoughts as well for the company name for sure but the copyright is still something I am not completely sure about.  @ChengJin01 can you make the change so that those 2 strings are defaulted to empty and allow consumers to set them.